src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 50px;
+  margin-right: 50px;
+  width: 100%;
+  align-self: center;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo {
+  display: flex;
+  text-decoration: none;
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+nav {
+  margin-right: 30px;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  margin-top: 22px;
+  margin-right: 20px;
+  height: 15px;
+  color: #000;
+  align-items: center;
+
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+li:last-child {
+  margin-right: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.